src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -178,6 +179,29 @@ public Collection<MergeReviewConceptVersions> getMergeReviewConflictingConcepts(
 		return conflicts.values();
 	}
 
+	private boolean isTargetConceptVersionBehind(Concept sourceVersion, Concept targetVersion) {
+		Set<Integer> sourceEffectiveTimes = new HashSet<>();
+		sourceEffectiveTimes.add(sourceVersion.getReleasedEffectiveTime());
+		sourceEffectiveTimes.addAll(sourceVersion.getRelationships().stream().map(Relationship::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getDescriptions().stream().map(Description::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getDescriptions().stream().map(Description::getAssociationTargetMembers).flatMap(List::stream).map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getAllOwlAxiomMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getInactivationIndicatorMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getAssociationTargetMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		while (sourceEffectiveTimes.remove(null)) ;
+
+		Set<Integer> targetEffectiveTimes = new HashSet<>();
+		targetEffectiveTimes.add(targetVersion.getReleasedEffectiveTime());
+		targetEffectiveTimes.addAll(targetVersion.getRelationships().stream().map(Relationship::getReleasedEffectiveTime).collect(Collectors.toList()));
+		targetEffectiveTimes.addAll(targetVersion.getDescriptions().stream().map(Description::getReleasedEffectiveTime).collect(Collectors.toList()));
+		targetEffectiveTimes.addAll(targetVersion.getDescriptions().stream().map(Description::getAssociationTargetMembers).flatMap(List::stream).map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		targetEffectiveTimes.addAll(targetVersion.getAllOwlAxiomMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		targetEffectiveTimes.addAll(targetVersion.getInactivationIndicatorMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		targetEffectiveTimes.addAll(targetVersion.getAssociationTargetMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+
+		return !targetEffectiveTimes.containsAll(sourceEffectiveTimes);
```

Comment:
Comparing released effective times will work for most of time but in certain cases this logic will not work. Especially  when international releases start releasing on the first day of every month. For example US edition 20230301 is dependant on international 20230131 release.  We start releasing on the first day of each month for international. After upgrading US extension to international 20230701, during rebase source and target might have same set of effective times however there might be changes made for international components in 20230301. This method will return false but in fact the target is a version behind.

Reply:
Wow, good edge case @CoderMChu! I'll work on improving this logic.

Reply:
I have updated the logic @CoderMChu. Now, only components in the International module will have their released effective time read. If the US only has 2022-01-31 International releasedEffectiveTimes, then they have diverged.

Reply:
@dmcgihtsdo Does this new API need to check extension content is a version behind? If it does then above fix won't work.

Reply:
It does need to check Extensions @CoderMChu. I'm not sure why you think it won't work; let's have a call?